_data/merchants.yml

```diff
@@ -214,6 +214,8 @@
       url: https://walletmonero.com
     - name: Web4Crypto - Website development
       url: https://web4crypto.xyz/
+    - name: Webbdo Hosting
```

Comment:
Maybe better add `(sw)` to warn people the website is in Swedish, like in [line 272](https://github.com/monero-project/monero-site/pull/623/files#diff-f9443b57eea572f90d7947d03f3113c3R272)

Reply:
Updated. Went with `(se)` instead of `(sw)`.